promptsource/templates/glue/mrpc/templates.yaml

```diff
@@ -1,35 +1,49 @@
-dataset: glue
-subset: mrpc
-templates:
-  3b88d2c4-0aeb-4c6d-9ccc-653a388250a5: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 3b88d2c4-0aeb-4c6d-9ccc-653a388250a5
-    jinja: '{% if label == 1 %}
-
+"dataset": |-
+  glue
+"subset": |-
+  mrpc
+"templates":
+  "3b88d2c4-0aeb-4c6d-9ccc-653a388250a5": !Template
+    "answer_choices": !!null |-
+      null
+    "answer_choices_key": !!null |-
+      null
+    "id": |-
+      3b88d2c4-0aeb-4c6d-9ccc-653a388250a5
+    "jinja": |-
+      {% if label == 1 %}
       Paraphrase the following sentence: {{sentence1}}
 
       |||
 
       {{sentence2}}
 
```

Comment:
The block scalar's literal style `|(-)` doesn't need single or double quotes, and will guarantee WYSIWYG for newlines.

If we want one the jinja to act in this manner, then we will need to modify more places like this: https://stackoverflow.com/a/20863889

Reply:
I really dislike the way the new format looks, it seems to break the nice parts of Yaml. I think we should try the more complex version to only change the jinja lines.

Reply:
I guess a third option is as https://github.com/bigscience-workshop/promptsource/pull/406#discussion_r707334059 mentioned that we simply don't enforce yaml format.